packages/components/src/Clickable/Clickable.module.css

```diff
@@ -147,14 +147,14 @@
 }
 
 .variantOutline {
-  background-color: var(--button-secondary-color);
+  background-color: transparent;
```

Comment:
Could also just remove the style since it's not overriding anything. But it does make it clear though that the buttons intentionally have a transparent background.